test-utils/src/main/kotlin/org/cqfn/save/testutils/MockWebServerUtils.kt

```diff
@@ -20,35 +20,35 @@ class LoggingQueueDispatcher : Dispatcher() {
     private var failFastResponse: MockResponse = MockResponse().setResponseCode(HttpURLConnection.HTTP_NOT_FOUND)
 
     private fun getProperRegexKey(path: String?, setOfRegexes: Iterable<String>) = path?.let {
```

Comment:
Let's may be write some tests for logic, corresponding with regex extraction
Thus, we will be sure, that similar requests will process properly and there will not some similar situations like in https://github.com/analysis-dev/save-cloud/issues/617